ledger/caching/src/test/lib/scala/com/daml/caching/MapBackedCacheForTesting.scala

```diff
@@ -17,4 +17,6 @@ final class MapBackedCacheForTesting[Key, Value](store: ConcurrentMap[Key, Value
 
   override def getOrAcquire(key: Key, acquire: Key => Value): Value =
     store.computeIfAbsent(key, acquire(_))
+
+  override def invalidate(key: Key): Unit = ()
```

Comment:
should not this `invalidate` actually delete the entry from the map?

Reply:
Missed this one, thanks 👍  Updated

Reply:
Dropped as not needed